exec/java-exec/src/main/java/org/apache/drill/exec/store/sys/store/LocalPersistentStore.java

```diff
@@ -61,16 +63,29 @@
   private final AutoCloseableLock readLock = new AutoCloseableLock(readWriteLock.readLock());
   private final AutoCloseableLock writeLock = new AutoCloseableLock(readWriteLock.writeLock());
 
+  //Provides a threshold above which we report the time to load
+  private static final long LISTTIME_THRESHOLD_MSEC = 2000L;
+
+  private static final int DrillSysFileExtSize = DRILL_SYS_FILE_SUFFIX.length();
```

Comment:
`DrillSysFileExtSize` -> `drillSysFileExtSize`

Reply:
I wanted to treat this like a constant, but this makes it confusing as a Class name